mods/bones/init.lua

```diff
@@ -186,18 +186,39 @@ minetest.register_on_dieplayer(function(player)
 		bones_mode = "bones"
 	end
 
+	local enable_bones_logging = minetest.settings:get_bool("enable_bones_logging")
+	local enable_bones_chat_msg = minetest.settings:get_bool("enable_bones_chat_msg")
+	local pos = vector.round(player:getpos())
+
 	-- return if keep inventory set or in creative mode
 	if bones_mode == "keep" or (creative and creative.is_enabled_for
 			and creative.is_enabled_for(player:get_player_name())) then
+		if enable_bones_logging then
+			minetest.log("action", "[bones] " .. player:get_player_name() ..
+				"'s bones do not remain since player has creative -- died at " ..
+				minetest.pos_to_string(vector.round(player:getpos())))
+		end
+		if enable_bones_chat_msg then
+			minetest.chat_send_player(player:get_player_name(), player:get_player_name() ..
+				"'s bones do not remain since player has creative -- died at " .. minetest.pos_to_string(pos))
```

Comment:
`pos` here too. Line is too long (more than 80 characters), please break it again.